dvc/utils/__init__.py

```diff
@@ -262,7 +262,14 @@ def fix_env(env=None):
             # NOTE: py2 doesn't like unicode strings in environ
             env[cast_bytes_py2(lp_key)] = cast_bytes_py2(lp_orig)
         else:
-            env.pop(lp_key, None)
+            # We used to delete LD_LIBRARY_PATH key. We cannot do that
+            # because GitPython uses env to update his own internal state.
+            # When there is no key in env, this value is not updated and
+            # GitPython re-uses LD_LIBRARY_PATH that has been set by
+            # PyInstaller.
```

Comment:
Let's use the same syntax we use elsewhere. Smth like
```suggestion
            # PyInstaller. See [1] for more info.
            # [1] https://github.com/gitpython-developers/GitPython/issues/924
```